apps/google-analytics-4/lambda/src/lib/verify-signed-request-middleware.ts

```diff
@@ -8,6 +8,7 @@ import express from 'express';
 import { InvalidSignature } from '../lib/errors/invalid-signature';
 import { UnableToVerifyRequest } from '../lib/errors/unable-to-verify-request';
 import { IncomingHttpHeaders } from 'http';
+import { HttpMethod } from 'googleapis-common';
```

Comment:
This might have been unintentional auto importing. I don't think we want to get the HttpMethod from the google apis lib though, it creates an unnecessary dependency between our  middleware and the fact that we happen to use google api libs elsewhere.

Reply:
ah gross, you're totally right :(

Reply:
proposal: since this app is _for google analytics_ is it reasonable to punt on this nit? i _absolutely_ get your point, but feel it's low value to redeclare this type interface _just in the name of "purity"_